core-blocks/heading/edit.native.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { RichText } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+const minHeight = 50;
+
+class HeadingEdit extends Component {
+	render() {
+		const {
+			attributes,
+			setAttributes,
+			style,
+			className,
+		} = this.props;
+
+		const {
+			align,
+			content,
+			level,
+			placeholder,
+		} = attributes;
+
+		const tagName = 'h' + level;
+
+		return (
+			<View>
+				<RichText
+					wrapperClassName="wp-block-heading"
```

Comment:
I don't think you will want to use `wrapperClassName` prop in here, I bet you rather convert it to style and pass down. @hypest should know better.

Reply:
Thanks for the ping @gziolo , I'd be inclined to keep using classnames instead of styles until the component that needs them transforms them to pass to the RN components that will consume them. That said, I can't say we know for sure yet since we haven't really started _actually_ incorporating styling.